content/extraProblems.json

```diff
@@ -705,6 +705,71 @@
         "usacoId": "1190"
       }
     },
+    {
+      "uniqueId": "usaco-1204",
+      "name": "Photoshoot 2",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1204",
+      "source": "Bronze",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Brute Force"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1204"
+      }
+    },
+    {
+      "uniqueId": "usaco-1205",
+      "name": "Blocks",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1205",
+      "source": "Bronze",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["Brute Force"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1205"
+      }
+    },
+    {
+      "uniqueId": "usaco-1207",
+      "name": "Robot Instructions",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1207",
+      "source": "Silver",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Recursion", "Map", "Sorting", "Meet in the Middle"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1207"
+      }
+    },
+    {
+      "uniqueId": "usaco-1208",
+      "name": "Email Filing",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1208",
+      "source": "Silver",
+      "difficulty": "Normal",
```

Comment:
note - this should be "Hard" if the test data was stronger. it is not that difficult to pass this problem with an incorrect solution

Reply:
lol is that why i got 8/12 test cases?